Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,19 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return tasks
+    .map(task => task.duration / 60)
```

Comment:
In this case it does not really matter. But in general it is advisable to do `.filter` before `.map`. Unless you really need the result of te map to do the filtering. Otherwise your code is wasting time mapping items it will filter out later.

Reply:
Okay Got it. I did these according to instructions order.